CHANGELOG.md

```diff
@@ -25,6 +25,7 @@ All notable changes to this project will be documented in this file based on the
 * support for elasticsearch-php ^7.0
 * Added `ParentAggregation` [#1616](https://github.com/ruflin/Elastica/pull/1616)
 * Elastica\Reindex missing options (script, remote, wait_for_completion, scroll...)
+* Added `AdjacencyMatrix` aggregation
```

Comment:
Could you add the PR number here?

Reply:
Yes, PR #1639 fixed the tests.
I pushed the smal change. :wink: